.ci/Jenkinsfile.gosh-prepare-rc

```diff
@@ -101,24 +101,53 @@ pipeline {
                 script {
                     sh "rm -rf tests_workspace_*"
                     def basePort = 9000
-                    def files = getTestFiles("tests")
+                    def files = getTestFiles("tests").toList()
                     def nodes = getNodesCount()
                     def versions = ["v1_x", "v2_x", "v3_x", "v4_x"]
-                    
-                    for (version in versions) {
-                        sh "rm -rf ${version}_workspace_*"
-                        deployVersion(version, nodes, basePort)
+
+                    def batchSize = 15
```

Comment:
I would be nice to define it as a constant somewhere at the top part of the file